### PR TITLE
Purecn/normaldb

### LINKED ISSUE #2939: new module: purecn/normaldb
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by aldosr**
## PR checklist

Closes #2939  

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by lbeltrame** [comment]
Blocked by https://github.com/BioContainers/multi-package-containers/pull/2554.

File: modules/nf-core/purecn/normaldb/meta.yml
Comment (nvnieuwk) [inline modules/nf-core/purecn/normaldb/meta.yml:1]: Can you use the same names as the output/input channels for the module as names in the meta? (e.g. `normal_panel` should be `normal_vcf`

**Review by nvnieuwk — COMMENTED** [review]
Very nice! Bonus points for adding the stub ;). Only some comments on the meta file

File: modules/nf-core/purecn/normaldb/meta.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - "purecn":
+      description: "Copy number calling and SNV classification using targeted short read sequencing"
+      homepage: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      documentation: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      tool_dev_url: "https://github.com/lima1/PureCN"
+      doi: "10.1186/s13029-016-0060-z"
+      licence: "Artistic-2.0"
+      args_id: "$args"
+
+input:
```
[the remark this answers is not in the document]
Comment (aldosr) [reply modules/nf-core/purecn/normaldb/meta.yml:22]: Forgot them, thanks :)

File: modules/nf-core/purecn/normaldb/meta.yml
Comment (aldosr) [reply modules/nf-core/purecn/normaldb/meta.yml:1]: We should be fine now :)

**Review by nvnieuwk — APPROVED** [review]
LGTM!

**Pull request merged**